test/e2e_node/e2e_service.go

```diff
@@ -252,6 +255,7 @@ func (es *e2eService) startKubeletServer() (*killCmd, error) {
 		"--file-check-frequency", "10s", // Check file frequently so tests won't wait too long
 		"--v", LOG_VERBOSITY_LEVEL, "--logtostderr",
 		"--pod-cidr=10.180.0.0/24", // Assign a fixed CIDR to the node because there is no node controller.
+		"--kubelet-cgroups=/kubelet",
```

Comment:
I modified my code to detect container names for docker and kubelet. Now it does not need to set the flag for CoreOS. I will update the PR for node performance tests.